src/bijectors/composed.jl

```diff
@@ -212,10 +212,10 @@ function forward(cb::Composed, x)
     
     for t in cb.ts[2:end]
         res = forward(t, rv)
```

Comment:
Same here, there's another `forward` above:
```suggestion
        rv, res_logjac = with_logabsdet_jacobian(t, rv)
```